docs/content/intro/nginx-ingress-controllers.md

```diff
@@ -17,7 +17,7 @@ If you are unsure about which implementation you are using, check the container
 
 ## The Key Differences
 
-The table below summarizes the key difference between nginxinc/kubernetes-ingress and kubernetes/ingress-nginx Ingress controllers. Note that the table has two columns for the nginxinc/kubernetes-ingress Ingress controller, as it can be used both with NGINX and NGINX Plus. For more information about nginxinc/kubernetes-ingress with NGINX Plus, read [here](nginx-plus.md).
+The table below summarizes the key difference between nginxinc/kubernetes-ingress and kubernetes/ingress-nginx Ingress controllers. Note that the table has two columns for the nginxinc/kubernetes-ingress Ingress controller, as it can be used both with NGINX and NGINX Plus. For more information about nginxinc/kubernetes-ingress with NGINX Plus, read [here](/nginx-ingress-controller/intro/nginx-plus).
```

Comment:
Recommend we rewrite the link so it's not just "here." That's poor style. It's a better UX to say what we're linking to.